lib/cfndsl/aws_types.yaml

```diff
@@ -640,6 +640,10 @@ Resources:
     Policies: [ IAMEmbeddedPolicy ]
    Attributes:
     Arn: String
+  "AWS::KinesisFirehose::DeliveryStream" :
+    Properties:
+      DeliveryStreamName: String
+      S3DestinationConfiguration: JSON
```

Comment:
Wouldn't this be better created as an S3DestinationConfiguration resource property Type?